src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -15,11 +15,14 @@
 #include "gna/gna_config.hpp"
 #include "gna_graph_tools.hpp"
 #include "gna_lib_ver_selector.hpp"
+#include "gna_limitations.hpp"
+#include "gna_plugin_config.hpp"
 #include "log/log.hpp"
 
 namespace ov {
 namespace intel_gna {
 namespace limitations {
+using namespace common;
```

Comment:
are you sure that it should be ov::intel_gna::limitations::common namespace?

Reply:
Good catch.